### DISCUSSION TITLE
Setup Lora SX126 output power to 30 dBm

### DISCUSSION THREAD

**Discussion by frogg1ven** [jgromes/RadioLib#291] [General]
Hi, 
in EBYTE Lora SX1262 specification, maximum output power equals 30 dBm, and in your library it can reach only 22 dBm. Is there way to change it to 30 dBm or is it impossible? 
I searched in the library files and all I found was this line:
`RADIOLIB_CHECK_RANGE(power, -17, 22, ERR_INVALID_OUTPUT_POWER);` in SX1262::setOutputPower function, but changing this value to 30 didn't work.
I know that to reach maximum Lora distance, I need to change Bandwidth to lowest, Spreading Factor to highest and output power to highest.
Do you have any sugestions?

**Comment by jgromes** [comment]
I suggest checking the other issue raised about this, #184 - all the answers are in there.

**Reply by frogg1ven** [reply]
thank you very much
